ccd-definition/CaseEventToComplexTypes/COMPLY_CAFCASS-cafcassDirections.json

```diff
@@ -0,0 +1,93 @@
+[
```

Comment:
could maybe move these 3 with a comply with directions folder when you merge the changes in that I made the other day

Reply:
merged your changes, the thing with adding them to comply is that they are different events 🤔 what do you reckon?

Reply:
That's a good point but they are all related as well which could be a reason to group them into one folder

Reply:
hmm so we group by event, technically they are all separate events. Hesitating about putting them in the same folder, so I am going to say no for now. Easy to address in another PR, so I am not too worried about the outcome. Everyone happy?

Reply:
I'm fine with that